lib/stringify.js

```diff
@@ -110,6 +110,16 @@ var TICK = '`';
 var TILDE = '~';
 var UNDERSCORE = '_';
 
+/**
+ * Check whether `character` is alphanumeric.
+ *
+ * @param {string} character - Single character to check.
+ * @return {boolean} - Whether `character` is alphanumeric.
+ */
+function isAlphanumeric(character) {
+    return /\w/.test(character) && character !== UNDERSCORE;
+}
```

Comment:
Should be `isAlphabetic(character) || isNumeric(character)` but that requires extracting these functions and characters from `parse.js` (out of scope of this PR).